src/main/java/org/csanchez/jenkins/plugins/kubernetes/pod/retention/Reaper.java

```diff
@@ -92,12 +92,18 @@ public static Reaper getInstance() {
 
     @Override
     public void onOnline(Computer c, TaskListener listener) throws IOException, InterruptedException {
```

Comment:
Noting merge conflict with https://github.com/jenkinsci/kubernetes-plugin/pull/1083/commits/b34d34a2f60bc3223e04b93c0168c806ffa4e833 in https://github.com/jenkinsci/kubernetes-plugin/pull/1083.